### PR TITLE
added basilcss to wappalyzer

### PR DESCRIPTION
**Pull request by daniel-bergmann**
Added basilcss to wappalyzer

### CODE REVIEW

File: src/technologies/b.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
+  "Basil.css": {
+    "cats": [
+      18
+    ],
+    "description": "The CSS framework made with the goal of bridging the gap between developers who like using UI frameworks and those who prefer styling apps without them.",
+    "icon": "basilcss.svg",
```
Comment (ceopeo) [inline src/technologies/b.json:7]: need 32x32

File: src/technologies/b.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 {
+  "Basil.css": {
+    "cats": [
+      18
+    ],
+    "description": "The CSS framework made with the goal of bridging the gap between developers who like using UI frameworks and those who prefer styling apps without them.",
```
Comment (ceopeo) [inline src/technologies/b.json:6]: Basil.css is a responsive and customisable UI framework.

File: src/technologies/b.json [hunk 3]
```diff
@@ -1,4 +1,13 @@
 {
+  "Basil.css": {
+    "cats": [
+      18
```
Comment (ceopeo) [inline src/technologies/b.json:4]: 66

File: src/technologies/b.json [hunk 4]
```diff
@@ -1,4 +1,13 @@
...
+    ],
+    "description": "The CSS framework made with the goal of bridging the gap between developers who like using UI frameworks and those who prefer styling apps without them.",
+    "icon": "basilcss.svg",
+    "website": "https://basilcss.com",
+    "npm": ["basilcss"]
```
Comment (ceopeo) [inline src/technologies/b.json:9]: What is this line for?

**Review by ceopeo — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!

- need examples
- and traces how can we define technology

File: src/technologies/b.json (hunk 3 above)
Comment (daniel-bergmann) [reply src/technologies/b.json:4]: fixed.

File: src/technologies/b.json (hunk 2 above)
Comment (daniel-bergmann) [reply src/technologies/b.json:6]: fixed.

File: src/technologies/b.json (hunk 1 above)
Comment (daniel-bergmann) [reply src/technologies/b.json:7]: added.

File: src/technologies/b.json (hunk 4 above)
Comment (daniel-bergmann) [reply src/technologies/b.json:9]: removed, propably just a brainfart.

**Comment by daniel-bergmann** [comment]
Thanks for the feedback @ceopeo, how does it look now?

**Review by ceopeo — COMMENTED** [review]
This is an example of how Tailwind CSS is detected
`"scriptSrc": "\\.tailwindcss(?:tailwind-config-cdn)?\\.(?:com|js)",`

**Comment by daniel-bergmann** [comment]
Hi there @ceopeo, thanks for your feedback!

I've added the tracers and tried to test it locally by visiting https://basilcss.com/ and https://daniel-bergmann.github.io/pomo-todo/

Do you have any other idea how to improve or should we plug it in?

**Comment by AliasIO** [comment]
Sorry, the [diff](https://github.com/wappalyzer/wappalyzer/pull/7894/files) is a bit of a mess. Would you mind rebasing or re-applying your changes?

**Comment by daniel-bergmann** [comment]
allrighty then, ill make a new pull request

**Pull request closed without merging**